pubspec.lock

```diff
@@ -324,6 +330,6 @@ packages:
       name: yaml
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.1.10"
+    version: "2.1.12"
 sdks:
-  dart: ">=1.17.0-dev.6.2 <2.0.0"
+  dart: ">=1.23.0-dev.0.0 <2.0.0"
```

Comment:
we shouldn't move the SDK up this high as that's beyond any stable version currently existing.